ansible_ai_connect_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -369,13 +369,12 @@ export const useChatbot = () => {
             method: "POST",
             headers: {
               "Content-Type": "application/json",
+              Accept: "application/json,text/event-stream",
```

Comment:
This was not needed when the UI was connected directly to the Chatbot service, but when it is connected to the streaming chat endpoint on the AI Connect service (<-- still under development), UI did not connect without this.